README.md

```diff
@@ -98,7 +98,7 @@ where `int` is the integrator used.
 - Preconditioning on metric space `metric`: `pc = Preconditioner(metric)`
 - Nesterov's dual averaging with target acceptance rate `δ` on integrator `int`: `da = NesterovDualAveraging(δ, int)`
 - Combine the two above *naively*: `NaiveHMCAdaptor(pc, da)`
-- Combine the first two using Stan's windowed adaptation: `StanHMCAdaptor(n_adapts, pc, da)` where `n_adapts` is the total number of adaptation steps will be used.
+- Combine the first two using Stan's windowed adaptation: `StanHMCAdaptor(pc, da)`
```

Comment:
Maybe add an example configuration for NUTS about how to 

1. fix mass-matrix, but adapt step size using `NesterovDualAveraging`
2. fix step size, but adapt mass-matrix using `Preconditioner`

Reply:
Are those just the two lines above?

Reply:
I see, thanks.